lib/webPush.js

```diff
@@ -137,6 +137,13 @@ export const notifyUserSubscribers = async ({ models, item }) => {
       AND "Reply"."ancestorUserId" = follower.id
       AND follower."noteAllDescendants"
     )
+    -- ignore subscription if user has posted to a territory the recipient is subscribed to
+    AND NOT EXISTS (
+      SELECT 1 FROM "SubSubcription"
+      INNER JOIN "Item" ON "Item"."subName" = "SubSubscription"."subName"
+      WHERE "SubSubscription"."userId" = "UserSubscription"."followerId"
+      AND "Item"."userId" = $1
+    )
     `, Number(item.userId), Number(item.id))
```

Comment:
I don't think you need this join, you can use `item.subName` as `$3` to get the territory and then use this filter:

```suggestion
      WHERE "SubSubscription"."userId" = "UserSubscription"."followerId"
      AND "SubSubscription"."subName" = $3
    )
    `, Number(item.userId), Number(item.id), item.subName)
```

Reply:
Added an isPost to $3 because comments don't have a subName